src/backend/src/modules/pia-intake/pia-intake.service.ts

```diff
@@ -171,7 +171,9 @@ export class PiaIntakeService {
     }
     if (query.filterByMinistry) {
       whereClause.forEach((clause) => {
-        clause.ministry = query.filterByMinistry;
+        clause.ministry = mpoMinistries.includes(query.filterByMinistry)
+          ? query.filterByMinistry
+          : IsNull();
```

Comment:
why IsNull ? it could simply be

if (condition) {
clause.ministry =query.filterByMinistry
}